app/controllers/user.js

```diff
@@ -0,0 +1,44 @@
+const bcrypt = require('bcrypt');
+const { User } = require('../models');
+const { signup } = require('../services/userService');
+
+const emailValidation = async email => {
+  const val = email.split('@');
+  const usr = await User.findAll({ where: { email } });
+  if (val[1] === 'wolox.com.ar' && usr.length === 0) {
+    return true;
+  }
+  return false;
+};
+
+const passwordValidation = pass => {
```

Comment:
you can also use express-validator to validate the password then use a helper to encrypt it